Dockerfile

```diff
@@ -25,6 +26,21 @@ RUN \
 
 ARG PYTHON_MATTER_SERVER
 
+ENV chip_example_url "https://github.com/agners/matter-linux-example-apps/releases/download/v1.3.0.0"
```

Comment:
can we move this to the libs org and at least target it into ota provide only ?

Reply:
> can we move this to the libs org and at least target it into ota provider only ?

Sure.

@frenck can you create a repo in the libs org? I'd name it `matter-linux-ota-provider`.

> Another option: include the final binary for the ota provider in the source code so the source code is at leats tied to a specific version of the ota provider

I am not a big fan in adding binaries into git. Also we'd then have to sort out architecture specifics there.

The Docker approach works nicely IMHO.

Reply:
Yeah, figured the fact that we distribute the Matter server as a whole (so the entire docker container/image) this is indeed fine. We should make the readme extra clear that a standalone install is not supported. I'll work a bit more on the docs after I get back from DC.

Reply:
Yeah if the binary is absent it will lead to an error when trying to update a device. We could preemptively check at startup and add a warning in the logs. But the update entities would still appear :thinking: 

I guess we could make it even a server info (`updates_supported`) and create  update entities on Core side only if the server does support updates. :thinking:

Reply:
Well, using the Matter server in HA is not supported from a manual install so this is a non-issue then.